doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-basic-operations-con.adoc

```diff
@@ -1,7 +1,7 @@
-[id='preview-editor-basic-operations-con']
-= Basic operations using the Test Scenario (Preview) editor
+[id='test-designer-basic-operations-con']
+= Basic operations using the Test Scenario designer
```

Comment:
Why `designer` now? I prefer `editor` personally (this comment is valid for the entire doc of course).

Reply:
As per a recent doc team's decisions, it was decided ti use the word *Designer* in place of "Editor" going forward. I believe Stetson has communicated this to Klara as well. Hence no change to this suggestion.